crates/rpc/src/handlers/version.rs

```diff
@@ -0,0 +1,33 @@
+use ream_node::version::ream_node_version;
+use serde::{Deserialize, Serialize};
+use warp::{
+    http::status::StatusCode,
+    reject::Rejection,
+    reply::{Reply, with_status},
+};
+
+use super::Data;
+
+#[derive(Serialize, Deserialize)]
+pub struct Version {
+    version: String,
+}
+
+impl Version {
+    pub fn new() -> Self {
+        Self {
+            version: ream_node_version(),
+        }
+    }
+}
+
+impl Default for Version {
+    fn default() -> Self {
+        Self::new()
+    }
+}
```

Comment:
```suggestion
#[derive(Serialize, Default)]
pub struct Version {
    version: String,
}

impl Version {
    pub fn new() -> Self {
        Self {
            version: ream_node_version(),
        }
    }
}
```

If you want Default you can derive it